src/app/docs/phone-field/phone-field-docs.component.ts

```diff
@@ -1,9 +1,29 @@
 import {
-  Component
+  Component,
+  OnInit
 } from '@angular/core';
 
+import {
+  FormControl,
+  FormGroup
+} from '@angular/forms';
+
 @Component({
   selector: 'app-phone-field-docs',
-  templateUrl: './phone-field-docs.component.html'
+  templateUrl: './phone-field-docs.component.html',
+  styleUrls: ['./phone-field-docs.component.scss']
 })
-export class PhoneFieldDocsComponent { }
+export class PhoneFieldDocsComponent implements OnInit {
+
+  public phoneForm: FormGroup;
+
+  public phoneControl: FormControl;
```

Comment:
Check your alphabetical ordering of properties.

Reply:
Done